amlab.js

```diff
@@ -0,0 +1,111 @@
+console.log('question1');
```

Comment:
Good job! It seems like you understand the array methods. Remember to use function syntax when answering your questions! Next time, push to try some of the bonuses! I'm pretty sure you can handle it!